src/main/java/ai/elimu/web/content/word/WordCsvExportController.java

```diff
@@ -71,6 +76,18 @@ public void handleRequest(
                 index++;
             }
             
+            JSONArray letterToAllophoneMappingsJsonArray = new JSONArray();
+            index = 0;
+            for (LetterToAllophoneMapping letterToAllophoneMapping : word.getLetterToAllophoneMappings()) {
+                JSONObject letterToAllophoneMappingJsonObject = new JSONObject();
+                letterToAllophoneMappingJsonObject.put("id", letterToAllophoneMapping.getId());
+                letterToAllophoneMappingJsonObject.put("letters", letterToAllophoneMapping.getLetters().stream().map(Letter::getText).collect(Collectors.joining()));
+                letterToAllophoneMappingJsonObject.put("allophones", letterToAllophoneMapping.getAllophones().stream().map(Allophone::getValueIpa).collect(Collectors.joining()));
```

Comment:
@Keerthi4308 Why does this generate 
```
"[{""allophones"":""k"",""id"":1,""letters"":""c""},{""allophones"":""æ"",""id"":2,""letters"":""a""},
{""allophones"":""t"",""id"":3,""letters"":""t""}]"
```
instead of 
```
"[{""letters"":""cat""},{""allophones"":""kæt""}]"
```

Is it because of `Collectors.joining()`?